src/CloudService/custom/New-AzCloudServiceLoadBalancerConfigurationObject.ps1

```diff
@@ -0,0 +1,48 @@
+
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the \"License\");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an \"AS IS\" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.Synopsis
+Create a in-memory object for LoadBalancerConfiguration
+.Description
+Create a in-memory object for LoadBalancerConfiguration
+
+.Outputs
+Microsoft.Azure.PowerShell.Cmdlets.CloudService.Models.Api20201001Preview.LoadBalancerConfiguration
+.Link
+https://docs.microsoft.com/en-us/powershell/module/az.CloudService/new-AzCloudServiceLoadBalancerConfigurationObject
+#>
```

Comment:
I am think if the model is very simple. Just like this one, there is only two members. Shall we still need to create a cmdlet for it

Reply:
Service team persist to keep it to provide a powershell way to create the object.